src/WPF/ArcGISRuntime.WPF.Viewer/Samples/MapView/DisplayGrid/readme.md

```diff
@@ -2,31 +2,35 @@
 
 Display coordinate system grids including Latitude/Longitude, MGRS, UTM and USNG on a map view. Also, toggle label visibility and change the color of grid lines and grid labels.
 
-![screenshot](DisplayGrid.jpg)
+![Image of display grid](DisplayGrid.jpg)
 
 ## Use case
 
 Grids are often used on printed maps, but can also be helpful on digital maps, to identify locations on a map.
 
+## How to use the sample
+
+Select type of grid from `Grid Type` (LatLong, MGRS, UTM and USNG) and modify its properties like label visibility, grid line color, and grid label color. Press the button to apply these settings.
```

Comment:
I think grid type should be in a different type of quote, or maybe removed altogether.

Reply:
Switched to plain english (no quotes), as it does not reference any standard or API enum